gitbook-docs/streaming_api.md

```diff
@@ -50,3 +50,31 @@ The server MAY provide:
 `name`, `self` and `roles` MUST return the same values as provided in the `swname`, `self` and `roles` properties within the [DNS-SD advertisement](connection.md) (if implemented).
 
 `version` MUST be the same value as `version` within the associated endpoints list provided by the http `GET` request to `/signalk` within the [REST API](rest_api.md) (if implemented).
+
+## History playback
+
+The server MAY support history playback from a certain point in time with a specified rate.
+
+To create a WebSocket connection that plays back data the client uses the request parameter `startTime` to specify the start timestamp and the optional request parameter `playbackRate` to specify the rate. Rate value parameter is a floating point value with value `1` equal to real time playback and for example `0.5` to half the real time rate and `5` to five times real time rate. Omitting the `playbackRate` will result in real time playback.
+
+Example url for history playback streaming: `wss://localhost:3443/signalk/v1/stream?subscribe=self&startTime=2018-08-24T15:19:09Z&playbackRate=5`.
```

Comment:
Why don't we make this a separate URL endpoint? This could be a `v1/playback`. This would make it very easy to detect if the server supports it.

Reply:
That would certainly be more explicit. Then again using the same url with extra parameters is pretty simple and straightforward imho.

@sbender9 ?

Reply:
For me, it makes the client code much simpler to keep the urls the same and add a parameter.

Reply:
I like having a separate endPoint. I currently don't really have a way to tell if the server supports this.